pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: value_layout_builder
 description: A LayoutBuilder with an extra value. It's useful when you want to build a widget with a value computed during layout.
-version: 0.2.0
+version: 0.3.0
```

Comment:
should we add `0.3.0-nullsafety.3` instead of `0.3.0`

Reply:
We can now publish stable release with nullsafety, so I think it's ok to set the new version to 0.3.0